public/app/core/components/TagFilter/TagOption.tsx

```diff
@@ -7,7 +7,7 @@ import { useStyles2 } from '@grafana/ui';
 
 import { TagBadge } from './TagBadge';
 
-export const TagOption = ({ data, className, label, isFocused, innerProps }: OptionProps<any, any>) => {
+export const TagOption = ({ data, className, label, isFocused, innerProps }: OptionProps<any, true>) => {
```

Comment:
what type is `true`? 🤔 have yet to see this, does this represent some type of union or?

Reply:
it's the type of `foo` here:
```ts
const foo = true;
```

i guess a specific narrowing of the boolean type? pretty sure in typescript they define `boolean` as `true | false`

although actually in this case maybe it would be safer to just omit the true and let it use the default `boolean` 🤔